packages/common/src/core/schemas/internal/metrics/interfaces.ts

```diff
@@ -22,4 +22,5 @@ export enum VISUAL_INTEREST {
 export enum SOURCE {
   dynamic = "dynamic",
   static = "static",
+  requesting = "requesting",
```

Comment:
do we want the underlying value of this option to be "requesting" or something more generic like "itemQery"? Or are we keeping the two separate? 🤔

Reply:
hmm...this is a good question. Perhaps we use itemQuery for now and when requesting metrics are implemented, we can update as needed.